index.html

```diff
@@ -1,20 +1,290 @@
+							<a href="#join" class=""> Join </a>
+						</li>
+
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#partner" class=""> Sponsor </a>
+						</li>
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#" class=""> News </a>
+						</li>
+						<li
+							class="border-2 border-coral-red py-2 px-4 transition-all hover:bg-coral-red hover:text-white hover:opacity-90">
+							<a href="index.html" class=""> CC Campaign </a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</nav>
+
+		<!-- Mobile menu -->
+		<div id="mobile-menu"
+			class="mobile-menu absolute top-0 hidden w-full origin-top animate-times-menu-animation flex-col justify-center bg-metal px-4 py-3 text-4xl text-white">
+			<button class="close self-start">X</button>
+			<div class="flex h-screen-94h flex-col items-center gap-4 py-8" aria-label="mobile-menu">
+				<a href="index.html" class="w-full nav-link px-4 text-center hover:opacity-70">Home</a>
+
+				<a href="./about.html" class="w-full nav-link px-4 text-center hover:opacity-70">About</a>
+				<a href="#program" class="nav-link w-full px-4 text-center hover:opacity-70">Program</a>
+				<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">Join</a>
+				<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">Sponsor</a>
+				<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">News</a>
+			</div>
+		</div>
+	</header>
+
+	<div
+		class="container relative h-full bg-cover bg-no-repeat hero-area mx-auto flex flex-col gap-4 px-4 py-28 text-metal-grey ">
+		<p class="text-2xl font-medium text-coral-red" style="color:#EC5242;">
+			"Hello! Sharing world"
+		</p>
+		<h1 class="back bg-cover bg-clip-text text-3xl font-bold uppercase text-transparent min-w-680px text-6xl">
+			World Conference <br />
+			Arabic Language Day <br />
+			2022
+		</h1>
+		<p class="border-4 border-white bg-coral-grey p-4">
+			A joyful celebration believing in the value of openness and
+			sharing, creating a positive change with people from over 80
+			countries is taking place in December, in Kuwait.
+		</p>
+
+		<div class="flex flex-col gap-3">
+			<h2 class="text-2xl font-bold">2022-12-14(WED) ~ 22(THU)</h2>
+			<h3 class="text-2xl font-medium">
+				&copy; Arabic Language Day Conference, KUWAIT CITY , KUWAIT
+			</h3>
+		</div>
+	</div>
+
+	<!-- main program section  -->
+	<section class="program" id="program">
+		<div>
+			<h2>Main Program</h2>
+			<img class="indicator" src="./public/images/indicator.svg" alt="Line Break" />
+		</div>
+
+		<div class="cards-container">
+			<!-- program card 1 -->
+			<div class="card-item">
+				<img src="./public/images/icon1.png" alt="Lecture Icon" />
+
+				<div class="activity">
+					<h3>Lecture</h3>
+				</div>
+				<div>
+					<p>
+						Listen to the speakers from various countries about
+						the messages of sharing and opening.
+					</p>
+				</div>
+			</div>
```

Comment:
- [ ] Kindly make sure to center the program paragraph in order to meet the design.
- [ ] Kindly increase the font size so your paragraph can be visible and easily readable.
- [ ] Kindly make sure the title looks the same as the provided one on the design.

    **_MAKE SURE TO APPLY THESE CHANGES TO EACH CARD_**

   - **YOUR DESIGN**
    ![image](https://user-images.githubusercontent.com/87197412/210710388-b52d028a-0e64-4625-b153-fca284d26fc1.png)

   - **PROVIDED DESIGN**
   
     ![image](https://user-images.githubusercontent.com/87197412/210710429-b52a7768-d29f-4dac-96fb-3497dc8811bb.png)